controllers/upgrade_handlers.go

```diff
@@ -30,7 +30,11 @@ func (r *ImageBasedUpgradeReconciler) handleUpgrade(ctx context.Context, ibu *ra
 	// pre upgrade steps
 	if err := r.ClusterConfig.FetchClusterConfig(ctx); err != nil {
```

Comment:
@donpenney Do you think the prep handler can make use of these or we really only need them for the upgrade stage?

Reply:
Generally we want to collect the data as close to the upgrade as possible, therefore we collect during the upgrade stage. The prep conceivably could be days before the actual upgrade